md/customize-living-application-layout.md

```diff
@@ -7,89 +7,12 @@
 ::: warning
```

Comment:
I think this page could be renamed Living application layout and should contain a short sentence explaining what is living application layout

Reply:
done